cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -2,18 +2,14 @@ name "Azure Hybrid Use Benefit for Windows Server"
 rs_pt_ver 20180301
 type "policy"
 short_description "Identifies eligible instances not utilizing Azure Hybrid Use Benefit.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/hybrid_use_benefit) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.2"
+long_description "Version: 1.3"
 severity "low"
 category "Cost"
 
-###############################################################################
-# Permissions
-###############################################################################
-
 permission "perm_instances" do
   label "Permissions for instances"
   actions "rs_cm.index", "rs_cm.show"
```

Comment:
Not sure why we need this, we aren't using rs_cm.instance resources.
However, we should have the credentials permissions block.

> permission "perm_read_creds" do
> actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
> resources "rs_cm.credentials"
> end

Reply:
Updated the permissions block.